### PR TITLE
combine consecutive filter plan nodes

### PR DESCRIPTION
**Pull request by czpmango**
Depends on [https://github.com/vesoft-inc/nebula-graph/pull/972](https://github.com/vesoft-inc/nebula-graph/pull/972)

### CODE REVIEW

**Comment by Aiee** [comment]
You can set some expected filters in ```operator info```

**Comment by czpmango** [comment]
> You can set some expected filters in `operator info`

ACK.

**Review by yixinglu — DISMISSED** [review]

**Review by Aiee — APPROVED** [review]

File: tests/tck/features/optimizer/CombineFilterRule.feature [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      """
+    Then the result should be, in any order:
+      | v                                                       | n                                                   |
+      | ("Steve Nash" :player{age: 45, name: "Steve Nash"})     | ("Jason Kidd" :player{age: 45, name: "Jason Kidd"}) |
+      | ("Vince Carter" :player{age: 42, name: "Vince Carter"}) | ("Jason Kidd" :player{age: 45, name: "Jason Kidd"}) |
+      | ("Jason Kidd" :player{age: 45, name: "Jason Kidd"})     | ("Steve Nash" :player{age: 45, name: "Steve Nash"}) |
+    And the execution plan should be:
+      | id | name         | dependencies | operator info                                                                                |
+      | 16 | Project      | 18           |                                                                                              |
+      | 18 | Filter       | 13           | {"condition" : "((($v.age>40) AND ($n.age>42)) AND (hasSameEdgeInPath($-.__COL_0)==false))"} |
```
Comment (yixinglu) [inline tests/tck/features/optimizer/CombineFilterRule.feature:25]: this expression should be updated since the pr #978

Comment (czpmango) [reply tests/tck/features/optimizer/CombineFilterRule.feature:25]: Get.

**Review by yixinglu — APPROVED** [review]

**Review by jievince — APPROVED** [review]
LGTM

**Pull request merged**